site/docs/how-to-release.md

```diff
@@ -308,6 +308,39 @@ Create a PR in the `iceberg` repo to make revapi run on the new release. For an
 - Create a PR in the `iceberg` repo to add the new version to the github issue template. For an example see [this PR](https://github.com/apache/iceberg/pull/6287).
 - Draft [a new release to update Github](https://github.com/apache/iceberg/releases/new) to show the latest release. A changelog can be generated automatically using Github.
 
+#### Update DOAP (ASF Project Description)
+
+- Create a PR to update the release version in `doap.rdf` file, in the `<release/>` section:
+
+```xml
+    <release>
+      <Version>
+        <name>x.y.z</name>
+        <created>yyyy-mm-dd</created>
+        <revision>x.y.z</revision>
+      </Version>
+    </release>
+```
+
+NB: we set only the last available release in the `doap.rdf` file. You can eventually list different versions, but we need to add multiple release sections:
```

Comment:
Maybe we don't have to confuse the readers by giving option, like arrow and other project, we can just keep the info about updating to the latest version.

Reply:
Above section we can mention that just update the version to latest release.

Reply:
OK, fair enough. Let me remove the Nota Bene 😄